packages/server-core/src/setting/authentication-setting/authentication-setting.hooks.ts

```diff
@@ -27,21 +27,158 @@ import { hooks as schemaHooks } from '@feathersjs/schema'
 import { iff, isProvider } from 'feathers-hooks-common'
 
 import {
+  AuthenticationSettingData,
+  AuthenticationSettingType,
   authenticationSettingDataValidator,
   authenticationSettingPatchValidator,
+  authenticationSettingPath,
   authenticationSettingQueryValidator
 } from '@etherealengine/engine/src/schemas/setting/authentication-setting.schema'
+import * as k8s from '@kubernetes/client-node'
 
+import { getState } from '@etherealengine/hyperflux'
+import { BadRequest } from '@feathersjs/errors'
+import { HookContext } from '../../../declarations'
+import logger from '../../ServerLogger'
+import { ServerState } from '../../ServerState'
+import config from '../../appconfig'
 import authenticate from '../../hooks/authenticate'
 import verifyScope from '../../hooks/verify-scope'
+import { AuthenticationSettingService } from './authentication-setting.class'
 import {
   authenticationSettingDataResolver,
   authenticationSettingExternalResolver,
   authenticationSettingPatchResolver,
   authenticationSettingQueryResolver,
-  authenticationSettingResolver
+  authenticationSettingResolver,
+  authenticationSettingSchemaToDb
 } from './authentication-setting.resolvers'
 
+/**
+ * Maps settings for admin
+ * @param context
+ * @returns
+ */
+const mapSettingsAdmin = async (context: HookContext<AuthenticationSettingService>) => {
+  const auth = (Array.isArray(context.result) ? context.result : [context.result]) as AuthenticationSettingType[]
```

Comment:
In which scenario `[context.result]` will be used? 

Also, how is Paginated result being handled?